Cargo.toml

```diff
@@ -570,3 +570,6 @@ required-features = ["remap-benches"]
 
 [patch.'https://github.com/tower-rs/tower']
 tower-layer = "0.3"
+
+[patch.crates-io]
```

Comment:
We should add a comment here explaining what we need to bump hyper at some point.

Reply:
Indeed - I'd also like to wait to merge this until we verify that it's the solution. I think based on our morning detective work it _is_ but if its at all possible to avoid merging an unnecessary crate patch that would be my preference.

Reply:
We can merge the patch and then switch to the released version. If we merge it the solution will be in the nightly and it'll be easier to test. We'll have to bump to publushed version either way, but we're imo in a good position to work with such patches.

I'll add a comment, and will merge if you agree.
We can already move from a fork to the upstream git since my PR was accepted.

Reply:
Sounds good to me!